src/ui/components/CommandPalette/CommandPalette.tsx

```diff
@@ -59,6 +65,12 @@ const COMMANDS: readonly Command[] = [
   { key: "show_privacy", label: "Show Privacy" },
   { key: "show_replay_info", label: "Show Replay Info" },
   { key: "show_sharing", label: "Show Sharing Options" },
+  { key: "toggle_dark_mode", label: "Toggle Dark Mode" },
```

Comment:
This might be better as `Set Appearance to Dark` , `Set Appearance to Light`, `Set Appearance to OS`